src/session.rs

```diff
@@ -42,6 +43,9 @@ pub struct Options {
     pub locale: String,
     pub scheme: AMQPScheme,
     pub properties: Table,
+
+    /// In seconds.
+    pub heartbeat: u16,
```

Comment:
Would it make sense for the `heartbeat` option to be a `Duration` ?

Reply:
Yea, I like that. `Option<Duration>`, per the discussion below.